tools/tools.go

```diff
@@ -1,3 +1,4 @@
+//go:build tools
```

Comment:
What does this do?

Reply:
Its for Go 1.17, its added automatically by `gofmt`